index.html

```diff
@@ -7,7 +7,82 @@
     <title>Vite App</title>
   </head>
   <body>
-    <div id="app"></div>
+    <div id="app">
+      <div>
+        <section>
+          <h1>Enter a comment</h1>
+        </section>
+        <section>
+            <form>
+              <div class="formDiv">
+                <label for="name">Name</label>
+                <br>
+                <input type="text" name="name" id="name" autocomplete="on" placeholder="Enter name..." required />
+              </div>
+              <div class="formDiv">
+                <label for="email">Email</label>
+                <br>
+                <input type="email" name="email" id="email" autocomplete="on" placeholder="Enter email..." required />
+              </div>
+              <div class="formDiv">
+                <label for="comment">Comment</label>
+                <br>
+                <textarea name="comment" id="comment" cols="30" rows="5" placeholder="Enter comment..." required ></textarea>
+              </div>
+              <span><input type="checkbox" id="checkbox" name="checkbox" value="checked" required>
+              </span><label id="agree">Agree to share info</label>
+              <br><br>
+              <input value="submit" type="submit" id="submit">
+              <div id="results-field">
+                <comment-com name="Kevin" email="fake@email.com" comment="seed some comments"></comment-com>
+                <comment-com name="Kevin" email="fake@email.com" comment="here"></comment-com>
```

Comment:
I added some comments here to make sure that the form will work fine if the page loads with some comments already loaded.

In practice, you will want to remove these, fetch the comments from your IndexedDB store, and create the list.
Something like: (didn't test this code, just as a rough idea)
```
dbPromise
  .then(function (db) {
    const tx = db.transaction('comments', 'readonly');
    const comments = tx.objectStore('comments');
    return comments.getAll();
  })
  .then(function (items) {
    const results = document.getElementById("results-field");
    items.forEach((item) => {
      const component = document.createElement('comment-com');
    
      component.setAttribute('name', item.name);
      component.setAttribute('email', item.email);
      component.setAttribute('comment', item.comment);
    
      const results = document.querySelector('#results-field');
      results.append(component);
  });
  ```